server/test/controllers/application/ApplicationTest.scala

```diff
@@ -40,7 +40,7 @@ class ApplicationTest extends PlaySpec with OneAppPerSuite {
     }
     "post form with missed fields to addTask" should {
       "result BadRequest with error1" in {
-        val Some(result) = route(FakeRequest(POST, "/addTask").withFormUrlEncodedBody("taskName" -> "0", "taskDescription" -> "1", "solutionTemplate" -> "2", "referenceSolution" -> "3" /*, "suite" -> "4"*/))
+        val Some(result) = route(app, FakeRequest(POST, "/addTask").withFormUrlEncodedBody("taskName" -> "0", "taskDescription" -> "1", "solutionTemplate" -> "2", "referenceSolution" -> "3" /*, "suite" -> "4"*/))
```

Comment:
there are some commented code in arguments. Let's get rid of any commented code in master.